Start of project1/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <figure>
+        <img src="images/octopus092118.jpg" alt="Lalala">
+        <figcation>
```

Comment:
Just a small typo in your element name here:
```suggestion
        <figcaption>
```

Also remember that an image + its caption is part of the requirement for the final project, so you'll need to add some text in the caption space here. The copy from lines 28 and 31 would make sense as captions for this photo.